app/src/main/java/com/lumisdinos/gameoffifteen/domain/usecases/DeleteGameUseCase.kt

```diff
@@ -0,0 +1,12 @@
+package com.lumisdinos.gameoffifteen.domain.usecases
+
+import com.lumisdinos.gameoffifteen.domain.repos.GameRepository
+import javax.inject.Inject
+
+class DeleteGameUseCase @Inject constructor(
+    private val repository: GameRepository
+) {
+    suspend fun executeUseCase(gameId: Int) {
+        return repository.deleteGame(gameId)
+    }
+}
```

Comment:
also you have another variant to write this (if you want):
suspend fun executeUseCase(gameId: Int) = repository.deleteGame(gameId)

Reply:
this variant more compact

Reply:
good point